python/tools/prepare_build_environment_linux.sh

```diff
@@ -29,7 +29,7 @@ else
         cuda-nvcc-12-2-12.2.140-1 \
         cuda-cudart-devel-12-2-12.2.140-1 \
         libcurand-devel-12-2-10.3.3.141-1 \
-        libcudnn8-devel-8.9.7.29-1.cuda12.2 \
+        libcudnn9-devel-cuda-12-9.1.0.70-1 \
```

Comment:
libcudnn9 from rhel7:

libcudnn9-devel-cuda-12-9.0.0.312-1

Reply:
RHEL7 has v9.0 as the highest available and it seems that they don't plan on releasing anything more updated, torch already uses 9.1.
Do you think the version we build with is irrelevant from the version users run with as long as they are the same major release? i.e. we can build with 9.0 and users can install 9.5 and get the new features, or are we limited by the version we build with?